scripts/python/get_afc_timing_delays

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env python3
+
+'''Script to get AFCs timing delays. Configure
+timing network to monitor events 115 and readback
+	Usage:
+    ./get_afc_timing_delays testbench
+	or
+		./get_afc_timing_delays sirius
```

Comment:
Nit: indentation

Reply:
Recently, I've been trying to use argparse instead of `sys.argv` and just allow it to define the help message. This script's option is simple enough that it isn't necessary, but I wanted to mention it.

Reply:
good to know, I will try argparse next time!